src/_docs/using.md

```diff
@@ -4,6 +4,8 @@ title: Using Albert
 permalink: /docs/using/
 ---
 
+To get up an running just run `albert` in terminal and press the hotkey (which is configured upon install, from the Settings menu item in the tray or in ~/.config/albert.config). To make albert autostart checkout [Troubleshooting](/docs/faq). 
```

Comment:
theres a typo (an/and). the mainstream way to launch apps is to start it from the menu since closing the terminal terminates the shell in it and therefore Albert too. manipulating the config file is discouraged since you can seriously break the integrity of the apps state.